tests/test_grid3d/test_grid_property.py

```diff
@@ -271,27 +271,20 @@ def test_pathlib(tmp_path, testdata_path):
         grdp.to_file(out, fformat="roff")
 
 
-def test_roffbin_import1(testdata_path):
-    """Test of import of ROFF binary"""
+@pytest.mark.parametrize("fformat", ["roff", "roffasc"])
+def test_roffbin_import1(testdata_path, fformat):
+    """Test of import of ROFF"""
 
     x = xtgeo.gridproperty_from_file(
-        testdata_path / TESTFILE1, fformat="roff", name="PORO"
+        testdata_path / TESTFILE1, fformat=fformat, name="PORO"
     )
 
     assert x.values.mean() == pytest.approx(0.1677, abs=0.001)
 
 
-def test_roffbin_import1_new(testdata_path):
-    """Test ROFF import"""
-
-    x = xtgeo.gridproperty_from_file(
-        testdata_path / TESTFILE1, fformat="roff", name="PORO"
-    )
-    assert x.values.mean() == pytest.approx(0.1677, abs=0.001)
```

Comment:
This was a duplicate of `test_roffbin_import1`.

Reply:
LGTM